### DISCUSSION TITLE
Login with email for login form

### DISCUSSION THREAD

**Discussion by digicali** [ExpressionEngine/ExpressionEngine#2931] [Feature Requests]
For some applications, a username doesn't make sense. Also, some people prefer to login with an email.
It will be great to have this option for login form.

**Comment by VarenSwaab** [comment]
I agree, it also makes for better options when using a social login service. I'd like to see this feature too.

**Comment by shbchk** [comment]
Giving only one choice (username and not email) is kind of outdated website behaviour, i agree. It needs updating, preferably along with the social sign-ins :)

**Comment by pyrobob83** [comment]
@intoeetive Robin said you were working on an overhaul to some members stuff... Any chance this could be considered for including in that?  I agree with shbchk that not being able to login with an email is outdated and we believe this is causing some confusion on client sites.

**Comment by intoeetive** [comment]
There you go: https://github.com/ExpressionEngine/ExpressionEngine/pull/3713

**Reply by intoeetive** [reply]
Getting this released in 7.4.0
